cli/boards.go

```diff
@@ -144,6 +177,37 @@ func (c *appClient) uploadBoardDefsFile(
 	return resp, nil
 }
 
+func (c *appClient) getBoardDefsFile(
+	name string,
+	version string,
+	orgID string,
+) (string, error) {
+	if err := c.ensureLoggedIn(); err != nil {
```

Comment:
I'm open to disagreement, but I would have put these 3 lines up in the top-level action, rather than down here. I think of this as part of validating that the command looks right, rather than part of getting the file itself.

Reply:
All other cli commands have this check here rather than the action function so wanted to keep consistent.

Reply:
Sigh. You're right that consistency is more important than doing what in my opinion is right. Leave it as-is.